hwbench/environment/vendors/pdus/tests/enlogic/outlet.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Could you document which endpoints should be called to get these files ? Maybe even provide a sample script to generate these files ?